templates/etc/systemd/system/keepalived.service.j2

```diff
@@ -1,3 +1,8 @@
+{% if keepalived_install_from == 'git' %}
+{% set keepalived_path = '/usr/local/sbin/' %}
+{% else %}
+{% set keepalived_path = '/usr/sbin/' %}
+{% endif %}
```

Comment:
Maybe move this logic to `vars` (as `keepalived_install_prefix`)?

Reply:
Fixed